ci/appveyor-cpp-build.bat

```diff
@@ -44,6 +44,7 @@ if "%JOB%" == "Static_Crt_Build" (
   cmake --build . --config Debug || exit /B
   ctest --output-on-failure -j2 || exit /B
   popd
+  rmdir /S /Q cpp\build-debug
```

Comment:
Because `find_package(gflags CONFIG)` can find GFlags in `cpp\build-debug\gflags_ep-prefix\`.